spec/models/pacbio/run_spec.rb

```diff
@@ -300,14 +300,12 @@
   end
 
   describe 'all_library_aliquots' do
-    it 'only returns aliquots with source_type Pacbio::Library' do
+    it 'returns all the aliquots used for a run' do
       library = create(:pacbio_library)
       pools = create_list(:pacbio_pool, 2)
       wells = [build(:pacbio_well, row: 'A', column: '1', libraries: [library]), build(:pacbio_well, row: 'B', column: '1', pools:)]
       run = create(:pacbio_revio_run, plates: [build(:pacbio_plate, wells:)])
-      run.all_library_aliquots.each do |aliquot|
-        expect(aliquot.source_type).to eq('Pacbio::Library')
-      end
+      expect(run.used_aliquots.size).to eq 8
```

Comment:
This test for `all_library_aliquots` can remain, I suppose. Otherwise, the `all_library_aliquots` method in the run model would be left untested. Alternatively, if it is not used anywhere, all_library_aliquots can be removed.

Reply:
I actually updated the all_library_aliquots method to used_aliquots as we need to publish for aliquots with source type = Pacbio::Pool too. 
This explains the updates test